### DISCUSSION TITLE
Trouble generating Dart code from Rust - no actual code being generated.

### DISCUSSION THREAD

**Discussion by shymega** [fzyzcjy/flutter_rust_bridge#914] [Q&A] [answered]
I'm working on a Flutter app, with a Rust backend.

I've been trying to setup this bridge, to generate Dart classes (as I understand it) from the Rust enums, which will soon be developed into something more featureful, but the problem I'm having so far is that I'm not getting *any* Dart code generated with the enum variants.

I couldn't see any IRC channel, or other chat service, so I'm opening a discussion.

Would someone be able to advise? The repos are for the backend: https://github.com/open-morphs/morph-rs and the app is on a branch on this repo: https://github.com/open-morphs/MorphInfiniControl/tree/ffi/initial-support

(Word of warning, rebasing is an active activity on this branch, which of course, wouldn't happen on mainline, but it's because I'm experimenting)

Thanks!

**Comment by fzyzcjy** [comment, marked as answer]
Well please create an issue, using the template, so that there are enough information to debug :)

**Reply by shymega** [reply]
Sure. I didn't want to create an issue, as I haven't yet confirmed on my end it is a bug or a problem with my code. Hence the discussion is opened. As a maintainer, I know issues can get quite full up, so I wanted to alleviate the burden.
